tests/common/runners/test_object_runner.py

```diff
@@ -0,0 +1,848 @@
+import unittest
+
+from checkov.common.runners.object_runner import Runner
+
+
+class TestObjectRunner(unittest.TestCase):
+    result = ({"name": "Supply Chain", True: {"workflow_dispatch": None,
```

Comment:
@NoaAzoulay Let the runner parse the file itself, to verify parser doesn't break please